packages/banner/ember-backstop/backstop_data/engine_scripts/puppet/overrideCSS.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-env browser, node */
```

Comment:
Can we do the same like backstop.js for this one also

Reply:
As these files are part of the addon, and since we intend to ensure a separation of concerns for each package, it would be ideal to have repetitive copies of the file.